Sources/AmazonLocationiOSAuthSDK/Types/ApiTypes/ResponseTypes/AmazonLocationBaseResponse.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+public class AmazonLocationBaseResponse {
```

Comment:
what's this one for? All base responses should have a status

Reply:
Removed it